src/qt/transactionrecord.cpp

```diff
@@ -92,6 +92,8 @@ QList<TransactionRecord> TransactionRecord::decomposeTransaction(const interface
             if(fAllToMe > mine) fAllToMe = mine;
         }
 
+        // MW: Single-line if to minimize diff
+        if (!wtx.txout_address.empty())
```

Comment:
I think it'd be better if you changed this to:

```
if (wtx.txout_address.empty())
{
    // Do nothing
}
else if (fAllFromMe && fAllToMe)
```

It still ends up with a minimal diff, but as it is right now, it's easy to miss that the whole if/else block is skipped when `wtx.txout_address` is empty.